### ISSUE TITLE
Chat platform

### ISSUE THREAD

**Issue by ishimotoshinobu** [Sneeds-Feed-and-Seed/sneedacity#49]
Sneedbros we need a chat platform, maybe matrix or xmpp. If you have e better ideas you can post them

**Comment by DevElonCoin** [comment]
if anything, use matrix

**Comment by qorg11** [comment]
#sneedacity in rizon?

**Comment by kakolisgay** [comment]
Claimed #sneedacity:matrix.org for the project. If some tards want to use irc or xmpp we can setup bridges for that

**Comment by Ckath** [comment]
I'd like to have an autism irc bridge the weechat plugin for matix still hurts

**Comment by eylles** [comment]
Isn't there some terminal matrix client?

**Comment by kakolisgay** [comment]
@eylles There's quickmedia (glorified cli client), gomuks, and wechat /w matrix plugin

**Comment by eylles** [comment]
Between gomuks and quickmedia what is the patrician option.

**Comment by kakolisgay** [comment]
gomuks if you want cli, quickmedia if you want a game engine pretending to be a chat client

**Comment by eylles** [comment]
> quickmedia if you want a game engine pretending to be a chat client

lolwut? don't sneed on me, quickmedia is a chucking game engine of all things?

**Comment by kakolisgay** [comment]
Quickmedia uses opengl for rendering the ui lol

**Comment by jobbautista9** [comment]
> I think the libera guys have a matrix bridge up already(?)

Yep, they already do. I have registered #sneedacity there just in case you guys want to bridge the matrix to irc.

**Comment by binex-dsk** [comment]
I set up an XMPP room `sneedacity@rooms.swirl.funny.cl`. don't recommend joining now unless you really want to, as i'll be migrating my XMPP stuff to a new server pretty soon

**Comment by kakolisgay** [comment]
PR #62 adds the matrix room and libera.chat room to the readme

**Comment by formerlychuck** [comment]
ok merged
